vignettes/predict-example.Rmd

```diff
@@ -2,7 +2,7 @@
 title: "Prediction using BCF"
```

Comment:
Why did you rename these files? I believe they were named this way to have them show up in a specific order

Reply:
vignettes can not start with a number (it won't pass the cran check and you won't be able to publish the package)

Reply:
https://github.com/r-lib/pkgdown/issues/545 

See commit message: https://github.com/ignacio82/bcf/commit/00d51a7c450418cb485a9c8e2e3957070e00530e